cmake/onnxruntime_training.cmake

```diff
@@ -102,7 +102,7 @@ if (onnxruntime_BUILD_UNIT_TESTS)
   onnxruntime_add_executable(onnxruntime_training_mnist ${training_mnist_src})
   onnxruntime_add_include_to_target(onnxruntime_training_mnist onnxruntime_common onnx onnx_proto ${PROTOBUF_LIB} onnxruntime_training flatbuffers)
   target_include_directories(onnxruntime_training_mnist PUBLIC ${CMAKE_CURRENT_BINARY_DIR} ${ONNXRUNTIME_ROOT} ${ORTTRAINING_ROOT} ${eigen_INCLUDE_DIRS} ${CXXOPTS} ${extra_includes} ${onnxruntime_graph_header} ${onnxruntime_exec_src_dir} ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_BINARY_DIR}/onnx onnxruntime_training_runner)
-
+  onnxruntime_add_include_to_target(onnxruntime_training_mnist Python::Module)
```

Comment:
Since we have introduced Python.h usage in framework package, and currently we used ENABLE_TRAINING flags for enabling the build of the kernels responsible for invoking Python calls. So By default, once we enable training, we will have Python.h dependency. Do you think we need introduce a new MACROs to control this new feature in build level?

Reply:
Get your point !